qlib/rl/order_execution/strategy.py

```diff
@@ -268,9 +515,8 @@ def _generate_trade_decision(self, execute_result: list = None) -> BaseTradeDeci
         act = policy_out.act.numpy() if torch.is_tensor(policy_out.act) else policy_out.act
         exec_vols = [self._action_interpreter.interpret(s, a) for s, a in zip(states, act)]
 
-        # In backtest, env.step() needs to be manually called since there is no outer trainer to call it
-        if self._backtest:
-            self._env.step(None)
+        self._state_interpreter.step()
```

Comment:
It is weird to help the interpreter maintain its state.
Will it be better to create a `cur_step` attribute in the state returned by the simulator?